### PR TITLE
Redesign menu & dialog – Main menu Closes #2685

### LINKED ISSUE #2685: Redesign menu & dialog – Main menu
Old:

New:

Spec:

Assets:

* [Number of trackers blocked](https://cl.ly/0R3e3N1N2G0j/trackers-blocked.svg)
* [Info](https://design.firefox.com/icons/viewer/#info)
* [Share…](https://design.firefox.com/icons/viewer/#share%20macos)
* [Home screen](https://design.firefox.com/icons/viewer/#home)
* [Open in…](https://design.firefox.com/icons/viewer/#open%20in%20new)
* [Request desktop site](https://design.firefox.com/icons/viewer/#desktop)
* [Settings](https://design.firefox.com/icons/viewer/#preferences)

Just a note that the icon fill follows our default value: `#f9f9fa` a80.

### PR DESCRIPTION
**Pull request by SrChip15**

### CODE REVIEW

**Comment by brampitoyo** [comment]
Refitted and recoloured icons:

* [Share…](https://cl.ly/2M3P232x342J/Share%20Macos.svg)
* [Home screen](https://cl.ly/2A290Y1Y1X31/Home.svg)
* [Open in…](https://cl.ly/2I0c3z1R1r0o/Open%20in%20New.svg)
* [Request desktop site](https://cl.ly/2v3S201T3m0U/Device%20Desktop.svg)
* [Settings](https://cl.ly/0E3F0I1c0J1j/Preferences%20Macos.svg)

**Comment by SrChip15** [comment]
@brampitoyo for some reason, the shared links (new) are opening up a blank page.

**Comment by SrChip15** [comment]
@brampitoyo Does this look right?


main_menu_rev

**Comment by SrChip15** [comment]
@brampitoyo Did you get time to look at this?

**Comment by brampitoyo** [comment]
@SrChip15 Sorry that this issue had fallen under my radar for so long!

> To confirm, the icons must be #f9f9fa . I do not understand what you mean by "a80" that follows the aforementioned color code.

It means that the icon should have 80% opacity. I think you can specify this in the code?

> The color of the trackers blocked box from the linked file is #b5007f. Is this correct?

This is correct.

**Comment by SrChip15** [comment]
@brampitoyo I have added the opacity. Take a final look and let me know if everything's fine.

**Comment by SrChip15** [comment]
@brampitoyo as requested, screenshot 👇

**Comment by brampitoyo** [comment]
Looks good. Thanks, @SrChip15!

**Comment by ekager** [comment]
Hi @SrChip15! Sorry for the wait. You'll have to add the license headers to the top of new vector xml files. Copy the top of another vector file. 

It would also be helpful if you rebased on the most recent master, resolved any conflicts, and removed any changes not related to this issue. It seems there are some other changes here. 

Thanks!

**Comment by SrChip15** [comment]
Hi @ekager! Not a problem at all. Ok, I'll add the license header and also sync up and rebase to keep only issue related changes. Appreciate your patience.

File: app/src/main/java/org/mozilla/focus/menu/browser/MenuItemViewHolder.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
         menuItemView = (TextView) itemView;
     }
-
+
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/menu/browser/MenuItemViewHolder.java]: \[Checkstyle\] ERROR: Trailing whitespace

**Comment by SrChip15** [comment]
Refer #3007

**Pull request closed without merging**